ocrd/ocrd/workspace.py

```diff
@@ -249,35 +249,46 @@ def _resolve_image_as_pil(self, image_url, coords=None):
         ]
         return Image.fromarray(region_cut)
 
-    def image_from_page(self, page, page_id, feature_selector='', feature_filter=''):
+    def image_from_page(self, page, page_id,
+                        fill='background', transparency=False,
+                        feature_selector='', feature_filter=''):
         """Extract a Page image from the workspace.
-
+        
         Given a PageType object, ``page``, extract its PIL.Image from
         AlternativeImage if it exists. Otherwise extract the PIL.Image
         from imageFilename. Also crop it if a Border exists, and rotate
         it if an @orientation exists. Otherwise just return it.
-
+        
         If ``feature_selector`` and/or ``feature_filter`` is given, then
         select/filter among imageFilename and all AlternativeImages the
         last which contains all of the selected but none of the filtered
         features (i.e. @comments classes), or raise an error.
-
+        
         If the chosen image does not have "cropped", but a Border exists,
         then crop it (unless "cropped" is also being filtered). And if the
         chosen image does not have "deskewed", but an @orientation exists,
         then rotate it (unless "deskewed" is also being filtered).
-
-        Cropping uses a polygon mask (not just the rectangle).
-
+        
+        Cropping uses a polygon mask (not just the rectangle). Areas outside
+        the polygon (regardless of cropping and deskewing) will be filled
+        according to ``fill``:
+        - if ``background`` (the default), then fill with the median color
+          of the image;
+        - if ``white``, then fill with white.
+        Moreover, ``transparency`` is true, then add an alpha channel which
+        is fully opaque before cropping and rotating (thus only the exposed
+        areas will be transparent afterwards, for those that can interpret
+        alpha channels).
+        
         (Required and produced features need not be in the same order, so
         ``feature_selector`` is merely a mask specifying Boolean AND, and
         ``feature_filter`` is merely a mask specifying Boolean OR.)
-
+        
         If the resulting page image is larger than the bounding box of
         ``page``, then in the returned bounding box, reduce the offset by
         half the width/height difference (so consumers being passed this
         image and offset will still crop relative to the original center).
-
+        
```

Comment:
We already [stumbled](https://github.com/OCR-D/core/pull/199#discussion_r228533807) over this: my editor prefers these indentations, probably because PEP 8 once recommended indented empty lines over blank lines. Also, the Python interpreter still requires interpretation, so when you c&p code...